src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1840,6 +1842,20 @@ input Location {
   place: String
 }
 
+"Maximum duration for access/egress for street searches per respective mode."
+input MaxAccessEgressInput {
+  bike: Duration
+  bikeRental: Duration
+  bikeToPark: Duration
+  car: Duration
+  carPickup: Duration
+  carRental: Duration
+  carToPark: Duration
+  flexible: Duration
+  scooterRental: Duration
+  walk: Duration
+}
```

Comment:
I think a list of streetmode - duration pairs would be a better format for these

Reply:
I've changed it now